### PR TITLE
Enketo joins ODK post

### PR DESCRIPTION
**Pull request by MartijnR**

### CODE REVIEW

File: _posts/2023-05-14-enketo-joins-odk.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Enketo joins ODK
```
Comment (yanokwa) [inline _posts/2023-05-14-enketo-joins-odk.md:2]: Enketo has joined ODK!

File: _posts/2023-05-14-enketo-joins-odk.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+author: Martijn van de Rijdt
+layout: post
+permalink: /enketo-joins-odk/
```
Comment (yanokwa) [inline _posts/2023-05-14-enketo-joins-odk.md:5]: /enketo-has-joined-odk/

File: _posts/2023-05-14-enketo-joins-odk.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Though I have had a great time developing Enketo and collaborating with so many impressive people, the past few years I have become increasingly interested in working on other things (such as woodworking). In August 2021, the ODK team agreed to take over leadership of the Enketo project. My own involvement in the greater Enketo project has reduced to just a few hours a week, and that work is mostly on custom features for a client.
+
+One and a half years after handing over control, it is clear that Enketo is in excellent hands! ODK (especially [Yaw Anokwa](https://www.linkedin.com/in/yanokwa), [Hélène Martin](https://forum.getodk.org/u/ln/summary), and [Trevor Schmidt](https://github.com/eyelidlessness)) have been doing an awesome job adding performance improvements, aligning behavior with ODK Collect, fixing critical vulnerabilities, and many other things. I am very impressed with their thoughtful approach to development (and am learning from it) and their willingness and ability to make big investments in improving the code, toolchain and maintainability. I am so happy that Enketo is continuing to thrive and am excited about its future.
```
Comment (yanokwa) [inline _posts/2023-05-14-enketo-joins-odk.md:18]: Add comma after toolchain to be consistent with previous comma usage.

File: _posts/2023-05-14-enketo-joins-odk.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Many thanks to ODK and all of Enketo's contributors over the years!
```
Comment (yanokwa) [inline _posts/2023-05-14-enketo-joins-odk.md:20]: I feel like we need a mild CTA of some sort. Maybe with a P.S.?

> P.S. If you haven't used ODK recently, you're missing out. Request a demo at [getodk.org](https://getodk.org) to learn more.

**Review by yanokwa — CHANGES_REQUESTED** [review]
It's looking pretty great to me. I've got some minor changes. Hoping we can ship this today!

Once it's done, we can merge https://github.com/enketo/enketo-website/pull/129.

**Comment by yanokwa** [comment]
Oh, and then once it's all merged, you can tweet it out? And we'll follow on with some tweets as well.

File: _posts/2023-05-14-enketo-joins-odk.md (hunk 1 above)
Comment (MartijnR) [reply _posts/2023-05-14-enketo-joins-odk.md:2]: Yes, very good!

**Comment by MartijnR** [comment]
> you can social media it out? 

Yes, I'll see if I can still write a tweet!

**Pull request merged**